cms/templates/settings.html

```diff
@@ -214,6 +214,16 @@ <h2 class="title-2">${_('Course Schedule')}</h2>
           </ol>
 
           <ol class="list-input">
+            <li class="field-group field-group-certificate-available" id="certificate-available">
+              <div class="field date" id="field-certificate-available-date">
+                <label for="certificate-available-date">${_("Certificates Available Date")}</label>
+                <input type="text" class="certificate-available-date date start datepicker" id="certificate-available-date" placeholder="MM/DD/YYYY" autocomplete="off" />
+                <span class="tip tip-stacked">${_("By default, 48 hours after course end date")}</span>
+              </div>
```

Comment:
@sanfordstudent Didn't we get in a big argument about this being "2 days" instead of "48 hours"? At any rate this is nice.